src/tiled/mainwindow.cpp

```diff
@@ -248,6 +248,9 @@ MainWindow::MainWindow(QWidget *parent, Qt::WindowFlags flags)
     mUi->menuMap->insertAction(mUi->actionOffsetMap,
                                mActionHandler->actionCropToSelection());
 
+    mUi->menuMap->insertAction(mUi->actionOffsetMap,
+                                mActionHandler->actionAutocrop());
```

Comment:
The indentation rule for wrapping function parameters is to align them to the opening `(`, so there is one space too many here.